content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -739,8 +739,6 @@ references it.
 
 ### Local
 
-{{< feature-state for_k8s_version="v1.10" state="beta" >}}
```

Comment:
If this feature is only stable in 1.14, should we call that out using an updated `feature-state` shortcode?
(I don't know when it was stabilized).